### PR TITLE
Added Rusyn keyboard layout.

### PR DESCRIPTION
**Pull request by novta**
Rusyn keyboard layout should be merged and published over AnySoftKeyboard organization.

### CODE REVIEW

File: addons/languages/rusyn/apk/build.gradle [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//owner unknown (may have not been published yet)
+ext.shouldBePublished = false
+
+apply from: "$rootDir/gradle/pack_apk.gradle"
```
Comment (menny) [inline addons/languages/rusyn/apk/build.gradle]: this should be: `apply from: "$rootDir/addons/gradle/pack_apk.gradle"`

File: addons/languages/rusyn/pack/build.gradle [hunk 2]
```diff
@@ -0,0 +1,6 @@
+ext.status_icon_text = "rue"
+
+ext.dictionaryInputPossibleCharacters = null//"".toCharArray()
+ext.dictionaryInputAdditionalInnerCharacters = null//"".toCharArray()
+
+apply from: "$rootDir/gradle/language_pack_lib.gradle"
```
Comment (menny) [inline addons/languages/rusyn/pack/build.gradle]: this should be `apply from: "$rootDir/addons/gradle/language_pack_lib.gradle"`

File: addons/languages/rusyn/pack/src/main/res/xml/rusyn_dictionaries.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Dictionaries>
+    <Dictionary
+        autoTextResourceId="@xml/rusyn_autotext"
+        description="@string/rusyn_dictionary_description"
+        dictionaryResourceId="@array/rusyn_words_dict_array"
+        id="a6cc6503-e08d-499d-9f0c-4471a67b3258"
```
Comment (menny) [inline addons/languages/rusyn/pack/src/main/res/xml/rusyn_dictionaries.xml]: this id is already used. Please generate a new one.

File: addons/languages/rusyn/apk/build.gradle (hunk 1 above)
Comment (novta) [reply addons/languages/rusyn/apk/build.gradle]: Fixed in commit bf9abef Resolved relative paths and unique id

File: addons/languages/rusyn/pack/src/main/res/xml/rusyn_dictionaries.xml (hunk 3 above)
Comment (novta) [reply addons/languages/rusyn/pack/src/main/res/xml/rusyn_dictionaries.xml]: Fixed in commit bf9abef Resolved relative paths and unique id

File: addons/languages/rusyn/pack/build.gradle (hunk 2 above)
Comment (novta) [reply addons/languages/rusyn/pack/build.gradle]: Fixed in commit bf9abef Resolved relative paths and unique id

**Comment by menny** [comment]
thank you

**Comment by novta** [comment]
Please, inform me about next release, with included Rusyn language kezboard layout.
I intend to inform my comunity about the new feature. 
Thank you

**Pull request merged**